examples/email-recipients/index.html

```diff
@@ -0,0 +1,88 @@
+<title>&lt;email&gt;</title>
+
+<link rel=stylesheet href=style.css>
+
+<email-recipients>
+
+  <template name=recipients>
+  <address>
+    <button id={#}
+      onclick=onremove
+      title='Remove <{self}> from recipients'>x</button>
+
+    <label>
+      <input name=recipients[] type=checkbox value={self} checked>
+      {self}
+    </label>
+  </address>
+  </template>
+
+  <input
+    type=search
+    name=address
+    list=contacts
+    onchange=addrecipient
+    placeholder=email@example.com>
+
+  <datalist id=contacts>
+  <template name=contacts>
+  <option value={email}>{name} &lt;{email}&gt;
+  </template>
+  </datalist>
+
+</email-recipients>
+
+<script src=/snuggsi.es></script>
+<script defer>
+
+Element `email-recipients`
+
+(class extends HTMLElement {
+
+  initialize () {
+    this.context.recipients =
+      ['angelo@isawesome.com', 'snuggs@isawesome.com']
+  }
+
+  onconnect () {
+    this.context
+      .contacts = [
+        { name: 'Tim', email: 'tim@isawesome.com' },
+        { name: 'Rob Cole', email: 'robcole@isawesome.com' }
+      ]
+  }
+
+  // use Array.prototype.splice
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice
+  onremove (event, id = event.target.id, address) {
+    address =
+      this.context
+        .recipients.splice
+          (event.target.id, 1)
+
+    this.select
+      `input[name=address]`.focus()
+  }
+
+  addrecipient (event, input = event.target) {
+    this.context
+      .recipients.push
+        (input.value)
+
+    input.value = ''
+  }
+
+  get recipients ()
+    { return this.context.recipients }
+
+  get contacts ()
+    { return this.context.contacts }
+})
+
+</script>
+
+<script src=/browser-sync.es></script>
```

Comment:
@brandondees we do already ;-) https://github.com/devpunks/snuggsi/blob/master/snippets/browser-sync.es#L3-L7